rocket-chat-android-widgets/src/main/java/chat/rocket/android/widget/message/MessageFormLayout.java

```diff
@@ -173,6 +208,61 @@ public void setEditTextCommitContentListener(
     this.listener = listener;
   }
 
+  public void setReplyContent(@NonNull AbsoluteUrl absoluteUrl, @NonNull Message message) {
+    String text = message.getMessage();
+    replyUsernameText.setText(message.getUser().getUsername());
```

Comment:
how hard would it be to add a check here and not add this if username is same as self?